docs/the-contribution-workflow.md

```diff
@@ -0,0 +1,245 @@
+## Contributing to Exercism
+
+Open source contributions can be intimidating! As you're looking around, wondering where to dive in, you might think everyone _else_ knows what's going on in the code base, or has better grasp of the language, or seems to be well known around the project.
+
+As a total beginner, where do you begin?
+
+Right here. This guide will walk you through your first open source contribution.
+
+It's going to target the smallest possible fix (improving documentation), as a "bridge" to more complex contributions later.
+
+First, go take a look at [Contributing to Exercism.io](https://github.com/exercism/exercism.io/blob/master/CONTRIBUTING.md#submitting-code-changes)
+
+If not all of that makes sense to you, keep reading. This is for you.
+
+
+## Table of Contents
+
+- [Find the right contribution](#find-the-right-contribution)
+- [Claiming that contribution](#claiming-that-contribution)
+- [Working on the code](#working-on-the-code)
+- [Submitting your contribution](#submitting-your-contribution)
+
+# Find the right contribution
+
+As mentioned in the contribution guideline, take a look at the [open issues](https://github.com/exercism/exercism.io/issues). There's a lot here, so you might want to filter by `label:"good first patch"` [(like this)](https://github.com/exercism/exercism.io/labels/good%20first%20patch).
```

Comment:
I would link to the good first patch label on the exercism org in general here:

https://github.com/search?utf8=%E2%9C%93&q=org%3Aexercism+is%3Aissue+is%3Aopen+label%3A%22good+first+patch%22&type=